extensions/wikia/ARecoveryEngine/ARecoveryBootstrapCode.class.php

```diff
@@ -0,0 +1,41 @@
+<?php
+class ARecoveryBootstrapCode {
+	public static function getHeadBootstrapCode() {
+
+		return (new ARecoveryModule())->isPageFairRecoveryDisabled() ?
```

Comment:
Can we base on method `enabled` instead of `disabled`?

+ can we make code more readable?
```
  if (!enabled()) {
    return 'coment';
  }

  return 'code';
```